code/_onclick/adjacent.dm

```diff
@@ -26,6 +26,9 @@
 		* Passing through in this case ignores anything with the throwpass flag, such as tables, racks, and morgue trays.
 */
 /turf/Adjacent(atom/neighbor, atom/target = null, list/ignore_list)
+	if(z != neighbor.z)
```

Comment:
```suggestion
	if(neighbor.z && (z != neighbor.z))
```
Atoms contained within other items - for example, anything currently worn or held by a human, or anything inside a storage - have z equal to 0. This seems to have broken a lot of things (trying to remove your webbing results in being told it is far away, trying to place a pistol into holster or backpack results in it firing, etc). This, or another adjacency check that I cannot find at a quick glace.